apps/src/sites/studio/pages/levels/editors/fields/_validation_code.js

```diff
@@ -12,7 +16,14 @@ function initPage() {
     widgetMode.on('click', () => syncValidateWithElements(embed, widgetMode));
   }
   if ($('#level_validation_code').length > 0) {
-    initializeCodeMirror('level_validation_code', 'javascript');
+    validationEditor = initializeCodeMirror(
+      'level_validation_code',
+      'javascript'
+    );
+  }
+  for (const name in data) {
+    const element = $('#generateValidation' + name);
+    element.on('click', () => validationEditor.getDoc().setValue(data[name]));
```

Comment:
based on the logic above, it seems like (when `$('#level_validation_code').length === 0`) that the validationEditor might not be initialized by the time we click. this is what I would recommend to avoid errors:
```suggestion
    element.on('click', () => validationEditor && validationEditor.getDoc().setValue(data[name]));
```

Reply:
@mikeharv looks like this one still needs to be addressed (maybe it was accidentally resolved before you saw it?)

Reply:
Oh, whoops! I actually committed your suggestion that directly through GitHub, but then I pulled `data[name]` out to a variable locally and essentially undid it when I resolved the merge conflict. Fixed again manually just now!